libraries/Microsoft.Bot.Builder/IRecognizer.cs

```diff
@@ -26,7 +26,7 @@ public interface IRecognizer
         /// <param name="context">Turn context.</param>
         /// <param name="ct">Cancellation token.</param>
```

Comment:
missed one in the docXML

Reply:
Thanks @JonathanFingold i needed to find-in-files again from the top!
So this should be good now.

Incidentally do you think IDialogContext context should be IDialogContext dialogContext ?

Open another issue if you think so. I'll close this one out now.